src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -263,6 +263,7 @@
   "skyblocker.config.general.itemTooltip.enableMuseumInfo.@Tooltip": "If this item is donatable to the museum, then the item's category in the musuem is displayed. It also displays a marker indicating whether you've donated that item to your musuem or not (freebies not yet supported).\n\nMake sure to enable your Museum API for accurate information!",
   "skyblocker.config.general.itemTooltip.enableNPCPrice": "Enable NPC Price",
   "skyblocker.config.general.itemTooltip.enableObtainedDate": "Enable Obtained Date",
+  "skyblocker.config.general.itemTooltip.showEssenceCost": "Show Essence Cost",
```

Comment:
Right now for all the price tooltips, it says "enable ...", but "show ..." is arguably better. So keep it as is?